enemies.py

```diff
@@ -215,6 +222,7 @@ def update_behavior(self, hero, bullets):
         if abs((hero.ground_position) - (self.ground_position)) < (hero.height * 0.25) and self.sprite in allTouching(
                 hero.sprite):
             changeSpriteImage(self.sprite, -2)
+            hero.health -= 20
```

Comment:
probably better to push ine one place all health cost of the different entities, so we have better control of heealth management